tests/e2e_regression/expected/emerald_contract_account.body

```diff
@@ -1,7 +1,826 @@
 {
-  "address": "oasis1qrrmuaed6numjju8gajzn68tn2edlvycjc50nfva",
+  "address": "oasis1qz2rynvcmrkwd57v00298uc2vtzgatde3cjpy72f",
```

Comment:
I changed the contract under test so we can capture all the verification info. (The old address is not verified.)